Library/Homebrew/cmd/update.sh

```diff
@@ -309,6 +314,11 @@ EOS
     set -x
   fi
 
+  if [[ -z "$HOMEBREW_AUTO_UPDATE_SECS" ]]
+  then
+    HOMEBREW_AUTO_UPDATE_SECS="60"
+  fi
```

Comment:
If you don't consider this too much voodoo witchcraft, the canonical (and much shorter) way to initialize variables with default values in Bash is to use expansion like this:

```shell
: "${HOMEBREW_AUTO_UPDATE_SECS:=60}"
```